docs/src/main/asciidoc/kafka-streams.adoc

```diff
@@ -1113,6 +1113,28 @@ CMD ["./application", "-Dquarkus.http.host=0.0.0.0", "-Xmx32m"]
 Now start Docker Compose as described above
 (don't forget to rebuild the container images).
 
+== Kafka Streams Health Check
+
+If you are using the `quarkus-smallrye-health` extension, `quarkus-kafka-streams` will automatically add a readiness health check to validate that all topics declared in the `quarkus.kafka-streams.topics` property are created.
+
+So when you access the `/health/ready` endpoint of your application you will have information about the available and missing topics.
+
+This is an example of `DOWN` status:
```

Comment:
Very cool idea to add the absence/presence of the input topics to the health check 👍 !

In [my post](https://quarkus.io/blog/kafka-streams-applications-with-quarkus-and-microprofile/#healthchecks) from some time ago I did something slightly different: querying the stream app state:

```
public HealthCheckResponse call() {
        return HealthCheckResponse.named("My Pipeline")
                .state(streams.state().isRunning())
                .build();
    }
```

Should we have both?

Reply:
I don't realy know if we should or not, the decision is yours. But we technically could.
Would you like I try to add the liveness of your blog post to get a view of the result ?

Reply:
I'd like to see the original check being added. It will tell you that the app is not healthy e.g. when receiving a poisonous message.

Reply:
Ok, I added your Liveness in addition to my Readiness. And I think the result is pretty good and meaningfull. Indeed if we only have the Liveness we don't really know why the streams is in `CREATED` state.

I also added the `streams.state()` in `data` of `HealthCheckResponse` to be more precise about why the streams is not running.

> Should we have both?

So my opinion is we really should have both ;)

http://localhost:9081/health
```
{
    "status": "DOWN",
    "checks": [
        {
            "name": "Kafka Streams state check",
            "status": "DOWN",
            "data": {
                "state": "CREATED"
            }
        },
        {
            "name": "Kafka Streams topic(s) check",
            "status": "DOWN",
            "data": {
                "Available topic(s)": "weather-stations,temperature-values",
                "Missing topic(s)": "missing-topic"
            }
        }
    ]
}
```

http://localhost:9081/health/ready
```
{
    "status": "DOWN",
    "checks": [
        {
            "name": "Kafka Streams topic(s) check",
            "status": "DOWN",
            "data": {
                "Available topic(s)": "weather-stations,temperature-values",
                "Missing topic(s)": "missing-topic"
            }
        }
    ]
}
```

http://localhost:9081/health/live
```
{
    "status": "DOWN",
    "checks": [
        {
            "name": "Kafka Streams state check",
            "status": "DOWN",
            "data": {
                "state": "CREATED"
            }
        }
    ]
}
```